source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,105 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+1. ssh into the logging machine and use the command line.  
+**Note:** You can probably do all of this with the OpenShift UI too - https://logging-machine:8443
```

Comment:
I thought that if I did not use ambiguity, the reader would expect detailed instructions to follow, which will not be soon forthcoming.  The instructions are for a (rare?) reader who is familiar with the OpenShift UI.

How about `https://fqdn.of.the.logging.machine:8443` or `The url of the OpenShift UI is 'https://hostname of the logging machine:8443'` or ???